### PR TITLE
platform section removal and other updates

### PR DESCRIPTION
**Pull request by hubvu**
**Main Notes**
- Deleted the **Platform** section, including the **OpenShift** **platform** page. 
  - Redirects will be required for the following 
    - https://docs.ondat.io/docs/platforms/ >>> https://docs.ondat.io/docs/install/
    - https://docs.ondat.io/docs/platforms/openshift/ >>> https://docs.ondat.io/docs/install/openshift/
  - the details in the openshift platform page contained mostly duplicate information that should actually fall under the prerequisites main page (changes are under away) 
- Removed duplicate/redundant information from the **Install** section - just like the openshift platform page, the details here should fall under the prerequisites page.
- Updated the description section for the **Install** section to a shorter, concise statement.
- Made small formatting, headers, typo fixes to the new **Upgrade** section and the documents.

### CODE REVIEW

File: docs/upgrade/upgrade.md [hunk 1]
```diff
@@ -1,39 +1,39 @@
 ---
 title: "Ondat Upgrade"
 linkTitle: Ondat Upgrade
+weight: 1
 ---
-# Overview
+## Overview
 
-This document details procedures to upgrade Ondat.
+This guide provides instructions how to upgrade Ondat.
```
Comment (cvlc) [inline docs/upgrade/upgrade.md:8]:
```suggestion
This guide provides instructions on how to upgrade Ondat.
```

File: docs/upgrade/upgrade.md [hunk 2]
```diff
@@ -1,39 +1,39 @@
...
 
-# Update an Ondat v2 cluster
+## Upgrading An Ondat `v2` Cluster
 
 ## Prerequisites
 
-> ⚠️ Ensure that you have read the [PIDs prerequisite introduced in Ondat v2.3](/docs/prerequisites/pidlimits) and that you check the init container logs to ensure your environments PID limits are set correctly.
+> ⚠️ Ensure that you have read the [PIDs prerequisite introduced in Ondat v2.3](/docs/prerequisites/pidlimits) and that you checked the init container logs to ensure your environments PID limits are set correctly.
```
Comment (cvlc) [inline docs/upgrade/upgrade.md:14]:
```suggestion
> ⚠️ Ensure that you have read the [PIDs prerequisite introduced in Ondat v2.3](/docs/prerequisites/pidlimits) and that you have checked the init container logs to ensure your environments PID limits are set correctly.
```

File: docs/upgrade/upgrade.md [hunk 3]
```diff
@@ -1,39 +1,39 @@
...
+> ⚠️ If you are upgrading to `v2.7.0`, you will only be able to downgrade to `v2.6.0` due to the mapping changes made in the Data Plane. For more information, review the [release notes](/docs/release-notes).
 
 ## Procedure
 
-### Step 1 - Backup of all Ondat yaml files
+### Step 1 - Backup Ondat Deployment Manifests
 
-* Make sure you keep a backup of all the Ondat yaml files.
-* You can also backup the Statefulset yaml files to keep track of the replicas.
+* Make sure you keep a backup of all the Ondat YAML files. You can also backup the `StatefulSet` yaml files to keep track of the replicas.
```
Comment (cvlc) [inline docs/upgrade/upgrade.md:26]:
```suggestion
* Make sure that you keep a backup of all the Ondat YAML files. You can also backup the `StatefulSet` yaml files to keep track of the replicas.
```

File: docs/upgrade/using-rolling-upgrades.md [hunk 4]
```diff
@@ -1,44 +1,47 @@
...
-# Prerequisites
+## Prerequisites
 
 > ⚠️ Make sure you have met the requirements of [configuring a Pod Disruption Budget (PDB)](https://kubernetes.io/docs/tasks/run-application/configure-pdb/).
 
 > ⚠️ If your volume does not have any replicas, the rolling upgrades feature will not start on any StorageOS node until you have one or more replicas on all your volumes.
 
 > ⚠️ This feature supports the following platforms: Google Anthos, Google GKE with future support to be expanded to Amazon EKS, Openshift and Rancher.
 
+> ⚠️ OpenShift requires the internal registry to be available during the upgrade, however Ondat volumes may not be available. Therefore using Ondat for the internal registry is not recommended.
```
Comment (cvlc) [inline docs/upgrade/using-rolling-upgrades.md:21]:
```suggestion
> ⚠️ Using Ondat for the internal registry is not recommended. OpenShift requires the internal registry to be available during the upgrade but Ondat volumes may become unavailable. 
```

File: docs/upgrade/using-rolling-upgrades.md [hunk 5]
```diff
@@ -1,44 +1,47 @@
...
 
-  ```
-  kubectl get storageoscluster -n storageos storageoscluster -o yaml | sed -e 's|^spec:$|spec:\n  nodeManagerFeatures:\n    upgradeGuard: "true"|' | kubectl apply -f - 
-  ```
+ ```bash
+ kubectl get storageoscluster -n storageos storageoscluster -o yaml | sed -e 's|^spec:$|spec:\n  nodeManagerFeatures:\n    upgradeGuard: "true"|' | kubectl apply -f - 
+ ```
 
-You will see new pods getting created, where one pod per node in a cluster called Node Manager.
+* You will see new pods getting created, where one pod per node in a cluster called Node Manager.
```
Comment (cvlc) [inline docs/upgrade/using-rolling-upgrades.md:42]:
```suggestion
* You will see new pods getting created, one pod per node in a cluster called Node Manager.
```

File: docs/upgrade/upgrade.md (hunk 2 above)
Comment (hubvu) [reply docs/upgrade/upgrade.md:14]: Thanks!

**Comment by hubvu** [comment]
Likewise, thanks for submitting the minor changes I have committed the changes you have suggested 💪  - the upgrade section does need more work, that will be another PR in the near future.

**Review by cvlc — APPROVED** [review]

**Pull request merged**